integration-cli/docker_cli_cp_test.go

```diff
@@ -472,7 +472,7 @@ func TestCpVolumePath(t *testing.T) {
 	}
 
 	cleanedContainerID := strings.TrimSpace(out)
-	defer deleteContainer(cleanedContainerID)
+	defer dockerCmd(t, "rm", "-fv", cleanedContainerID)
```

Comment:
-f calls `container.Stop(3)` which does the TERM & KILL after 3 seconds, deleteContainer does just a SIGKILL